api/src/main/java/org/openmrs/api/OrderExceptions.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api;
+
+import static org.openmrs.Order.Action.DISCONTINUE;
+
+/**
+ * Represents specific error conditions that are part of OrderService.
+ * The specific error classes define only a default constructor if they are meant to be used with only a single error message.
+ * In other cases, there should be static factory methods that can be used to create the error class with the specific error message needed.
+ */
+public class OrderExceptions {
```

Comment:
@samshuster do you mind throwing some more light as to why you prefer creating a new OrderExceptions class for grouping purposes instead of just putting them in the org.openmrs.api or any other package, as the rest of the existing custom exception classes? :)